nix/hm-module.nix

```
@@ -0,0 +1,81 @@
+{ pkgs, config, lib, ... }:
+
+with lib;
+
+let cfg = config.programs.hellwal;
+hellwal= pkgs.callPackage ./default.nix {};
+in {
+  options = { programs.hellwal = { enable = mkEnableOption "hellwal"; }; };
+
+  config = mkIf cfg.enable {
+
+    home.packages = [ hellwal ];
+    
+    programs.zsh.initExtra = ''
+      # Import colorscheme from 'hellwal' asynchronously
+      # &   # Run the process in the background.
+      # ( ) # Hide shell job control messages.
+      (cat ${config.xdg.cacheHome}/hellwal/sequences &)
+    '';
```

Hellwal is not generating sequences, but variables.sh and terminal.sh

```suggestion
    programs.zsh.initExtra = ''
      # Source variables
      source ${config.xdg.cacheHome}/hellwal/variables.sh
      # Printf ANSI escape codes
      sh ${config.xdg.cacheHome}/hellwal/terminal.sh
    '';

    programs.bash.bashrcExtra = ''
      # Source variables
      source ${config.xdg.cacheHome}/hellwal/variables.sh
      # Printf ANSI escape codes
      sh ${config.xdg.cacheHome}/hellwal/terminal.sh
    '';
```

---

As I said "Yet the module is merely a copy of pywal's module. Maybe you can tweak it yourself for better use cases"